### PR TITLE
Add support for logrus logging and port existing code.

### PR DESCRIPTION
**Pull request by dinowernli**
@hatstand

### CODE REVIEW

File: coordinator/coordinator.go [hunk 1]
```diff
@@ -2,12 +2,12 @@ package coordinator
 
 import (
 	"fmt"
-	"log"
 
 	"dinowernli.me/faucet/config"
 	pb_config "dinowernli.me/faucet/proto/config"
 	pb_worker "dinowernli.me/faucet/proto/service/worker"
 
+	logrus "github.com/Sirupsen/logrus"
```
Comment (hatstand) [inline coordinator/coordinator.go]: Remove redundant import name?

File: WORKSPACE [hunk 2]
```diff
@@ -20,3 +20,9 @@ http_archive(
     strip_prefix = "bazel-0.4.1",
     url = "https://github.com/bazelbuild/bazel/archive/0.4.1.tar.gz",
 )
+
+new_go_repository(
+    name = "com_github_sirupsen_logrus",
+    importpath = "github.com/Sirupsen/logrus",
+    commit = "d26492970760ca5d33129d2d799e34be5c4782eb",
```
Comment (hatstand) [inline WORKSPACE]: Use tag instead?

**Review by hatstand — APPROVED** [review]

File: coordinator/coordinator.go (hunk 1 above)
Comment (dinowernli) [reply coordinator/coordinator.go]: Done. Here and everywhere.

File: WORKSPACE (hunk 2 above)
Comment (dinowernli) [reply WORKSPACE]: Done.

**Pull request merged**